### ISSUE TITLE
Ssd disks

### ISSUE THREAD

**Issue by Hansdevries** [clinthuffman/PAL#50]
Hi

In threshold html page after generating a report a see that value of disks is about 5400 rpm is there a way to adjust this or use ssd disks?

**Comment by clinthuffman** [comment]
Yes, you can edit the threshold in the PAL Threshold Editor by selecting Quick System Overview as the threshold file and clicking Edit. It’s under \LogicalDisk(*)\Avg. Disk sec/Transfer. Make sure you do Update Threshold and do a File, Save. If you installed PAL in the default C:\Program Files\PAL folder, then you may need to save the threshold file to your Desktop folder, then manually copy/overwrite it back to the PAL installation folder, then restart the PAL wizard to see the change.

Keep in mind that the threshold is in place to indicate when there is significant delays in disk IO processing which can still happen on SSDs, but certainly less often. What I’m saying is that I use the same threshold for when I’m testing SSDs. There are some *really* bad SSDs on the market that are slower than 5400 RPM spindle disks.

Thank you,

________________________________

**Comment by leafar8710** [comment]
Hello,

Regarding the disk counter for both reading and writing, threshold is shown for 5400 RPM disks but I would like to know what is the threshold for 7200 RPM, 10000 RPM disks and for SSD, I am very grateful if you indicate how I do to determine it.

**Comment by jeffstokes72** [comment]
https://mcpmag.com/articles/2011/05/12/how-to-speak-san-ish.aspx?m=1

**Comment by jeffstokes72** [comment]
Was going to attach the chart in that but it's not a graphic
